src/prefect/server/schemas/actions.py

```diff
@@ -67,7 +67,7 @@ def validate_variable_name(value):
 
 class ActionBaseModel(PrefectBaseModel):
     class Config:
-        extra = "forbid"
+        extra = "ignore"
```

Comment:
In a test scenario for this, if we (or a user) provides an optional field but it's misspelled for example, like 
 `FlowRunUpdate`, let's say, they provide like `paramters` (where `parameters` is an optional field on `FlowRunUpdate`, how would they know they misspelled it if at all?

Reply:
yeah... that's a good point